src/undo/durable_move.py

```
@@ -0,0 +1,681 @@
+            return
+        except (ImportError, OSError):
+            # ``ImportError``: fcntl unavailable (shouldn't happen on
+            # POSIX but defensive). ``OSError`` on open can happen if
+            # the journal disappeared between ``exists`` and
+            # ``open`` — fall through to the unlocked path which
+            # handles the missing-file case.
+            pass
+    _sweep_unlocked_body(journal)
+
+
+def _sweep_locked_body(journal: Path, fh) -> None:  # type: ignore[no-untyped-def]
```

Ruff (0.15.11)

[warning] Unused function argument: `journal`

(ARG001)